content/modules/rest/pages/entities-api/load-entities.adoc

```diff
@@ -148,119 +171,114 @@ sort:: an entity attribute that will be used for sorting `(String)`.
+
+
+[source, http request]
+.Request
+----
+GET http://localhost:8080/rest
+            /entities
+            /sample_Order?sort=+date,-amount
+----
+
+[source, json]
+.Response: HTTP 200 - OK
+----
+[
+  {
     "id": "41aae331-b46b-85ee-b0bc-2de8cbf1ab86",
-    "date": "2021-02-02",
-    "amount": 283.55,
-    "createdDate": "2021-02-06T12:04:37.372",
-    "createdBy": "admin",
-    "lastModifiedDate": "2021-02-06T12:04:37.372",
-    "lines": [
-      {
-        "_entityName": "rstex11_OrderLine",
-        "_instanceName": "rest.sample.entity.OrderLine-d89df652-9d06-8d3e-2f26-5387e96e5b5d [detached]",
-        "id": "d89df652-9d06-8d3e-2f26-5387e96e5b5d",
-        "product": {
-          "_entityName": "rstex11_Product",
-          "_instanceName": "Cotek Battery Charger",
-          "id": "1ed85c7a-89f1-c339-a738-16307ed6003a",
-          "name": "Cotek Battery Charger"
-        },
-        "quantity": 1.0,
-        "createdDate": "2021-02-06T12:04:37.372",
-        "createdBy": "admin",
-        "lastModifiedDate": "2021-02-06T12:04:37.372",
-        "version": 1
-      },
-      {
-        "_entityName": "rstex11_OrderLine",
-        "_instanceName": "rest.sample.entity.OrderLine-7ccd45ec-8878-2a4a-2c24-b5ac053b6d4c [detached]",
-        "id": "7ccd45ec-8878-2a4a-2c24-b5ac053b6d4c",
-        "product": {
-          "_entityName": "rstex11_Product",
-          "_instanceName": "Solar-One HUP Flooded Battery 48V",
-          "id": "1860904a-5444-9c3e-9dc1-1d7a26d9ac19",
-          "name": "Solar-One HUP Flooded Battery 48V"
-        },
-        "quantity": 1.0,
-        "createdDate": "2021-02-06T12:04:37.372",
-        "createdBy": "admin",
-        "lastModifiedDate": "2021-02-06T12:04:37.372",
-        "version": 1
-      }
-    ],
-    "version": 1,
-    "customer": {
-      "_entityName": "rstex11_Customer",
-      "_instanceName": "Shelby Robinson",
-      "id": "bd1c8e90-3d35-cbe2-9efd-167202c758d2",
-      "name": "Shelby Robinson",
-      "version": 1
-    }
+    "date": "2021-02-02", // <1>
+    "amount": 283.55
+  },
+  {
+    "id": "288a5d75-f06f-d150-9b70-efee1272b96c",
+    "date": "2021-03-01",
+    "amount": 249.99, // <2>
+    "lastModifiedBy": "admin"
+  },
+  {
+    "id": "1068c217-5868-faf4-16aa-23655e9492da",
+    "date": "2021-03-01",
+    "amount": 130.08
   }
 ]
 ----
+<1> the result with the oldest date is returned first
+<2> when the `date` attribute is the same, the `amount` is used to sort the results
 
-NOTE: Every entity in the response has a `_entityName` attribute with the name of the entity, and an `_instanceName` attribute with the xref:data-model:entities.adoc#instance-name[Instance Name] of the entity.
+=== Using Pagination
+
+The Entities API support Pagination to respect the data processing limitation that might be present on the server or client-side. In case you want to load only a particular subset of the entities, you can provide the URL parameters `offset` and `limit`.
```

Comment:
"supports"

Also, about "you can provide the URL parameters `offset` and `limit`": you consistently use similar word composition, but it sounds not very good (not only my opinion, I've validated it).

It should be either with colon: "you can provide the URL parameters: `offset` and `limit`", 
or better in different order: "you can provide the `offset` and `limit` URL parameters".

Please keep it in mind in further writing.

Reply:
I'm not 100% on what you mean by similar word compositions. Is it about this very example or more in general?

I have changed the order accordingly. Will merge it for now. If you have further examples I can try to change it down the road.